doc/releases/release-notes-3.7.rst

```diff
@@ -91,6 +91,22 @@ Bluetooth
   * Removed ``err`` from :c:struct:`bt_bap_broadcast_assistant_cb.recv_state_removed` as it was
     redundant.
 
+  * Renamed ``stream_lang`` to just ``lang`` to better fit with the assigned numbers document.
+    This affects the ``BT_AUDIO_METADATA_TYPE_LANG`` macro and the following functions:
+    - :c:func:`bt_audio_codec_cap_meta_set_lang`
+    - :c:func:`bt_audio_codec_cap_meta_get_lang`
+    - :c:func:`bt_audio_codec_cfg_meta_set_lang`
+    - :c:func:`bt_audio_codec_cfg_meta_get_lang`
+
+  * Changed ``lang`` from ``uint32_t`` to ``uint8_t [3]``. This modifies the following functions
+    - :c:func:`bt_audio_codec_cap_meta_set_lang`
+    - :c:func:`bt_audio_codec_cap_meta_get_lang`
+    - :c:func:`bt_audio_codec_cfg_meta_set_lang`
+    - :c:func:`bt_audio_codec_cfg_meta_get_lang`
+
+    The result of this is that string values such as ``"eng"`` and ``"deu"`` can now be used to set
```

Comment:
Would it make sense to add a link to a list of the language codes?

Reply:
Not in the release notes IMO, but makes sense to have in the documentation. 

I've added a link to the values at the definition of `BT_AUDIO_METADATA_TYPE_LANG`.